core/src/test/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionValidatorSpec.scala

```diff
@@ -21,38 +21,54 @@ class ExpressionValidatorSpec extends FlatSpec with TestDriverComponent with Exp
     methodConfiguration.inputs.map(toMethodInput).toSeq
   }
 
-  val allValid = MethodConfiguration("dsde", "methodConfigValidExprs", "Sample", prerequisites=Map.empty,
+  val allValid = MethodConfiguration("dsde", "methodConfigValidExprs", Some("Sample"), prerequisites=Map.empty,
```

Comment:
same deal here: some real, some meaningless. `allowRootEntity = true` is the default so you can ignore that and only pay attention to changes with `allowRootEntity = false`